setup.cfg

```diff
@@ -53,7 +53,6 @@ console_scripts =
 dev =
     ghga-service-chassis-lib[dev]==0.16.1
```

Comment:
`chassis lib[dev]` could probably be deleted now.

Reply:
I think we would still need it as this is defining all the dev dependencies for mypy and co.